src/gcp/firestore.ts

```diff
@@ -36,12 +34,12 @@ var _listCollectionIds = function(project) {
  * @param {object} doc a Document object to delete.
  * @return {Promise} a promise for the delete operation.
  */
-var _deleteDocument = function(doc) {
+export async function deleteDocument(doc: any): Promise<void> {
```

Comment:
I think you want `Promise<any>` since I am pretty sure `api.request` will not return `void`

Reply:
done